src/NRediSearch/Client.cs

```diff
@@ -674,6 +997,92 @@ public Document GetDocument(string docId)
         public async Task<Document> GetDocumentAsync(string docId)
             => Document.Parse(docId, await _db.ExecuteAsync("FT.GET", _boxedIndexName, docId).ConfigureAwait(false));
 
+        /// <summary>
+        /// Gets a series of documents from the index.
+        /// </summary>
+        /// <param name="docIds">The document IDs to retrieve.</param>
+        /// <returns>The documents stored in the index. If the document does not exist, null is returned in the list.</returns>
+        public Document[] GetDocuments(params string[] docIds)
+        {
+            if (docIds.Length == 0)
+            {
+                return new Document[] { };
+            }
+
+            var args = new List<object>
+            {
+                _boxedIndexName
+            };
+
+            foreach (var docId in docIds)
+            {
+                args.Add(docId);
+            }
+
+            var queryResults = (RedisResult[])DbSync.Execute("FT.MGET", args);
+
+            var result = new Document[docIds.Length];
```

Comment:
perhaps special-case zero length result with `Array.Empty<Document>()`

Reply:
At the top of this method I'm checking to see if the length of the `docIds` array is 0, if so then I'm returning an `Array.Empty<Document>()` (I just changed that it was `new Document[]{}`. 

The intent here is to return `null` for any document not retrieved.